2.6/Dockerfile

```diff
@@ -1,10 +1,19 @@
 FROM centos:centos7
-MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
+LABEL MAINTAINER SoftwareCollections.org <sclorg@redhat.com>
```

Comment:
I would put this under the same LABEL command as the other labels. Also use the key="value" format that is used for existing labels.